### PR TITLE
Create HelplessPeasants

### PR DESCRIPTION
**Pull request by vadimjprokopev**

### CODE REVIEW

File: server/src/game/cards/00-human/common/UnitHelplessPeasants.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import CardColor from '@shared/enums/CardColor'
+import CardFaction from '@shared/enums/CardFaction'
+import GameEventType from '@shared/enums/GameEventType'
+import CardLocation from '@shared/enums/CardLocation'
+import ExpansionSet from '@shared/enums/ExpansionSet'
+import CardFeature from '@shared/enums/CardFeature'
+import CardTribe from '@shared/enums/CardTribe'
+import BuffStrength from '../../../buffs/BuffStrength'
+import BuffProtector from '@src/game/buffs/BuffProtector'
+
```
Comment (Tenebrie) [inline server/src/game/cards/00-human/common/UnitHelplessPeasants.ts:13]:
```suggestion
+ /* Original design and implementation by Nenl */
```

File: server/src/game/cards/00-human/common/UnitHelplessPeasants.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			.getUnitsOwnedByPlayer(thisUnit.owner)
+			.some(
+				(anotherUnit) =>
+					this.game.board.getDistanceToStaticFront(anotherUnit.rowIndex) < unitDistance &&
+					this.game.board.getHorizontalUnitDistance(thisUnit, anotherUnit) < 1 &&
+					anotherUnit.card.buffs.has(BuffProtector)
+			)
+
+		if (isProtected) {
+			thisUnit.buffs.add(BuffStrength, this)
```
Comment (Tenebrie) [inline server/src/game/cards/00-human/common/UnitHelplessPeasants.ts:53]: Want to make that scalable? As recurring buff potency, probably.

File: server/src/game/cards/00-human/common/UnitHelplessPeasants.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+		this.createCallback(GameEventType.TURN_ENDED, [CardLocation.BOARD])
+			.require(({ player }) => player === this.ownerInGame)
+			.perform(() => this.onTurnEnded())
+	}
+
+	private onTurnEnded(): void {
+		const thisUnit = this.unit!
+		const unitDistance = this.game.board.getDistanceToStaticFront(thisUnit.rowIndex)
+		console.log(unitDistance)
```
Comment (Tenebrie) [inline server/src/game/cards/00-human/common/UnitHelplessPeasants.ts:42]: Console.log!

File: server/src/game/cards/00-human/common/UnitHelplessPeasants.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import BuffProtector from '@src/game/buffs/BuffProtector'
+
+export default class UnitHelplessPeasants extends ServerCard {
+	constructor(game: ServerGame) {
+		super(game, {
+			type: CardType.UNIT,
+			color: CardColor.BRONZE,
+			faction: CardFaction.HUMAN,
+			tribes: [CardTribe.PEASANT],
+			features: [CardFeature.KEYWORD_DEPLOY, CardFeature.PROTECTOR],
```
Comment (Tenebrie) [inline server/src/game/cards/00-human/common/UnitHelplessPeasants.ts:21]:
```suggestion
			features: [CardFeature.KEYWORD_DEPLOY, CardFeature.KEYWORD_PROTECTOR],
```

File: server/src/game/cards/00-human/common/UnitHelplessPeasants.ts (hunk 2 above)
Comment (vadimjprokopev) [reply server/src/game/cards/00-human/common/UnitHelplessPeasants.ts]: Done.

**Comment by vadimjprokopev** [comment]
👍

**Review by Tenebrie — APPROVED** [review]
🚢 it

**Pull request merged**